CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## Unreleased
 
+## 0.19.1 (February 27th, 2022)
+
+CHANGES:
```

Comment:
We'll update the changelog release date at release time, so please leave this out for now.

```suggestion
CHANGES:
```

Reply:
Make sense indeed :)
Was not sure about the process. I reverted the changelog change.